.gitignore

```diff
@@ -15,3 +15,6 @@ astroid.egg-info/
 .pytest_cache/
 .mypy_cache/
 venv
+.venv/
```

Comment:
```suggestion
```
This is specific to your dev env and should be in a global gitignore on your side. (You won't have to add this in all your project)

Reply:
Ah apologies, yes. I've removed this and put it in my global `.gitignore`.